contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/minimize/minimize.css

```diff
@@ -1,6 +1,6 @@
 .tui-toolbar-btn-minimize {
   float: right !important;
-  background: url('./toolbar-icon-minimize.svg');
+  background: url('./toolbar-icon-minimize.svg') !important;
```

Comment:
Is `!important` really needed? Seems to work for me without it. The same applies to other plugins.

Reply:
Ah yes, that was something that was an issue with v1 and not in v2 anymore